tcms/testplans/static/testplans/js/get.js

```diff
@@ -210,6 +210,9 @@ function drawStatusAndPriority() {
             liElement.find('a').text(testCaseStatus.name);
             liElement.find('a')[0].dataset.id = testCaseStatus.id;
             listItemsNodes.push(liElement);
+
+            //add statuses to the toolbar dropdown
+            $('.js-toolbar-status').append(liElement);
```

Comment:
This has the side effect of adding the menu items on the bulk-menu and removing them from the TestCase menu. Same for priority.

Also note: these menu items are currently missing event handlers but the handlers will be different in the future. I don't think that would play well if these are the same object.

OTOH all statuses and priorities are available when the view is rendered so they can be passed in the context and rendered via regular Django template for loops. B/c these are used in several places, most notably the rows with test cases which are dynamic you can also use the `<template>` HTML tag to render the sub-menu once and then clone the DOM nodes and attach them to where necessary.